docs/guides/hanadb/concepts/catalog.md

```
@@ -0,0 +1,60 @@
+title: HanaDBVersion CRD
+menu:
+  docs_{{ .version }}:
+    identifier: hanadb-catalog-concepts
+    name: HanaDBVersion
+    parent: hanadb-concepts
+    weight: 15
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)